### PR TITLE
DolphinWX/DolphinQt2 CMakeLists: Remove use of LIBS and SRCS variables

### PR DESCRIPTION
**Pull request by lioncash**
Instead, operate on the target itself.

### CODE REVIEW

File: Source/Core/DolphinQt2/CMakeLists.txt [hunk 1]
```diff
@@ -120,19 +120,20 @@ set(SRCS
   TAS/StickWidget.cpp
   TAS/IRWidget.cpp
   Updater.cpp
+  ${UI_HEADERS}
```
Comment (degasus) [inline Source/Core/DolphinQt2/CMakeLists.txt]: Where is UI_HEADERS defined? I can't find it :/

**Comment by degasus** [comment]
LGTM

File: Source/Core/DolphinQt2/CMakeLists.txt (hunk 1 above)
Comment (leoetlino) [reply Source/Core/DolphinQt2/CMakeLists.txt]: Looks like it's just unused because we don't have any calls to `qt5_wrap_ui`.

Comment (lioncash) [reply Source/Core/DolphinQt2/CMakeLists.txt]: Removed it

**Pull request merged**